controllers/snapshot/snapshot_adapter.go

```diff
@@ -251,7 +256,7 @@ TestScenarioLoop:
 				"snapshot", a.snapshot.Name,
 				"environment.Name", copyEnv.Name,
 				"snapshot.Spec.Components", a.snapshot.Spec.Components)
-			return reconciler.RequeueWithError(err)
```

Comment:
Why this is not error anymore?

Reply:
Comparing with `RequeueWithError`, `reconciler.RequeueAfter(requeueDelay, err)` is having an additional delay with requeue.

Reply:
Removed the manual delay since we can't implement it currently due to the issue in https://github.com/redhat-appstudio/operator-goodies/blob/main/reconciler/handler.go#L27.